src/protobuf_impl.rs

```diff
@@ -103,7 +103,11 @@ impl Builder {
             desc.get_file(),
             &files_to_generate,
             &Path::new(&self.out_dir),
-            &protobuf_codegen::Customize::default(),
+            &protobuf_codegen::Customize {
+                carllerche_bytes_for_bytes: Some(true),
```

Comment:
How about using extension instead?

Reply:
`lite_runtime` is a different option in `Customize`. Here is the code:
```rust
let lite_runtime = rustproto::exts::lite_runtime_all.get(source);
Customize {
    lite_runtime,
    // carllerche_bytes_for_bytes is not set.
    ..Default::default(),
}
```